src/pages/DownloadMonitor.vue

```diff
@@ -19,18 +19,50 @@
                         <div class="border-at-bottom pad pad--sides">
                             <div class="card is-shadowless">
                                 <p><strong>{{ downloadObject.initialMods.join(", ") }}</strong></p>
-                                <p v-if="downloadObject.progress < 100">Downloading: {{ downloadObject.modName }}</p>
-                                <p>{{Math.min(Math.floor(downloadObject.progress), 100)}}% complete</p>
-                                <Progress v-if="!downloadObject.failed"
-                                    :max='100'
-                                    :value='downloadObject.progress'
-                                    :className="['is-info']"
-                                />
-                                <Progress v-else-if="downloadObject.failed"
-                                    :max='100'
-                                    :value='100'
-                                    :className="['is-danger']"
-                                />
+
+                                <div v-if="downloadObject.failed">
+                                    <p>Download failed</p>
+                                    <Progress
+                                        :max='100'
+                                        :value='100'
+                                        :className="['is-danger']"
+                                    />
+                                </div>
+
+                                <div v-else class="row">
+
+                                    <div class="col">
+                                        <p v-if="downloadObject.downloadProgress < 100">Downloading: {{ downloadObject.modName }}</p>
+                                        <p v-else>Downloading:</p>
+                                        <p>{{Math.min(Math.floor(downloadObject.downloadProgress), 100)}}% complete</p>
```

Comment:
To match with "Download failed", this could also simply say "Download complete" on one row without the percentage, once both download and installation is done.

Reply:
I'm probably not the right person to judge whether the completed progress bar should be green or stay blue. But changing the failed download bar to green is mixed signals and should IMO be avoided.